plugins/authentication/ldap/ldap.php

```diff
@@ -85,7 +85,7 @@ public function onUserAuthenticate($credentials, $options, &$response)
 
 					$ldap->bind($dn, $this->params->get('password', ''));
 				}
-				catch (ConnectionException $exception)
+				catch (ConnectionException|LdapException $exception)
```

Comment:
```suggestion
				catch (ConnectionException | LdapException $exception)
```
PHPCS complains now about missing spaces around the "|". No idea why it hasn't complained before.